### DISCUSSION TITLE
Welcome to downlords-faf-client Discussions!

### DISCUSSION THREAD

**Discussion by Sheikah45** [FAForever/downlords-faf-client#2363] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by micheljung** [comment]
Hi I'm Downlord and I put plenty of bugs into the client so I can watch you fixing them 😜

**Reply by Sheikah45** [reply]
Thank you for the contributions to my learning experience 😂

**Comment by Brutus5000** [comment]
Hi I'm Brutus and I get constantly pinged for reviews, even though I have no clue about the codebase.

**Reply by Sheikah45** [reply]
Your years of experience are enough

**Comment by bukajsytlos** [comment]
Am I right here? In a group dedicated to old grumpy devs?

**Reply by Brutus5000** [reply]
You are as grumpy and old as you feel 🤭

**Comment by 1-alex98** [comment]
Hi I am alex I have not done anything in ages but I am watching you :D
